server/cmwell-sparql-agent/src/main/scala/cmwell/tools/data/sparql/SparqlProcessorManager.scala

```diff
@@ -227,105 +234,84 @@ class SparqlProcessorManager (settings: SparqlProcessorManagerSettings) extends
     }
   }
 
-/*
+
   /**
     * Generates data for tables in cm-well monitor page
     */
-  def stringifyActiveJobs(activeJobs: Map[String, JobStatus], currentNonActiveConfigs: Configs): Future[Iterable[Table]] = {
+  def stringifyActiveJobs(jobs: Jobs): Future[Iterable[Table]] = {
     implicit val timeout = Timeout(1.minute)
 
-    def generateNonActiveTables(currentNonActiveConfigs: Configs) = currentNonActiveConfigs.map { case (path, config) =>
-      val sensorNames = config.sensors.map(_.name)
-      val title = Seq(s"""<span style="color:red"> **Non-Active** </span> ${path.utf8String}""")
-      val header = Seq("sensor", "point-in-time")
+    def generateNonActiveTables(jobs: Jobs) = jobs.collect { case (path, jobStatus@(_: JobPaused | _:JobFailed)) =>
 
-      val req = HttpRequest(uri = s"http://${settings.hostConfigFile}${path.utf8String}/tokens?op=stream&recursive&format=ntriples&fields=token")
-
-      // get stored tokens
-      val storedTokensFuture: Future[Map[String, Token]] = Source.single(req -> blank)
-        .via(httpPool)
-        .mapAsync(1) {
-          case (Success(HttpResponse(s, _, e, _)), _) if s.isSuccess() =>
-            e.withoutSizeLimit().dataBytes.runFold(blank)(_ ++ _)
-          case (Success(HttpResponse(s, _, e, _)), _) =>
-            logger.error(s"cannot get stored tokens for non-active jobs: status=$s, entity=$e")
-            e.discardBytes()
-            Future.successful(blank)
-          case (Failure(err), _) =>
-            logger.error(s"cannot get stored tokens for non-active jobs: err=$err")
-            Future.successful(blank)
-        }.runWith(Sink.head)
-        .map { rawData =>
-          rawData.utf8String.split("\n").map(_.split(" "))
-            .collect { case Array(s, p, o, _) =>
-              val token = if (o.startsWith("\"")) o.init.tail else o
-              val sensorName = Paths.get(s).getFileName.toString.init
-              sensorName -> token
-            }.toMap
-        }
+      val sensorNames = jobStatus.job.config.sensors.map(_.name)
+      val title = Seq(s"""<span style="color:red"> **Non-Active - ${jobStatus.statusString} ** </span> ${path}""")
```

Comment:
Paused job is not a failure it merely a job the user stopped. It shouldn't be red.